vespa-cloud/album-recommendation-searcher/src/main/application/services.xml

```diff
@@ -24,20 +23,20 @@
              - Reference: https://docs.vespa.ai/en/reference/services-container.html#document-api
              - Operations: https://docs.vespa.ai/en/document-api.html
         -->
-        <document-api />
+        <document-api/>
 
         <!--
-            <search /> tells the container to answers queries and serve results for those queries.
-            Inside the <search /> cluster you can configure "search chains" - a Java plugin that
-            will do chained processing of both queries and results.
+            <search> tells the container to answers queries and serve results for those queries.
+            Inside the <search /> cluster you can configure chains of "searchers" -
+            Java components processing the query and/or result.
 
             See:
              - Reference: https://docs.vespa.ai/en/search-api.html
              - Searchers: https://docs.vespa.ai/en/searcher-development.html
         -->
         <search>
             <chain id="metalchain" inherits="vespa">
-                <searcher id="ai.vespa.example.album.MetalSearcher" bundle="albums" after="MinimalQueryInserter">
```

Comment:
I think we had to add this to make YQL searches work - maybe this has changed since

Reply:
Yes, not changed but should be. But it's already set in the searcher itself.